common/app/navigation/Navigation.scala

```diff
@@ -170,7 +170,12 @@ object NavMenu {
       val childrenToShow = if (currentNavHasChildren) currentNavLink else currentParent
       val children = childrenToShow.map(navLink => navLink.children).getOrElse(Nil)
 
-      Subnav(parent, children)
+      // TODO remove after world cup as a bit of a hack
```

Comment:
You can also achieve this by putting `"football/world-cup-2018",` [here](https://github.com/guardian/frontend/blob/master/common/app/navigation/Navigation.scala#L128) and [here](https://github.com/guardian/frontend/blob/master/common/app/navigation/NavLinks.scala#L596).

Given this is temporary, I am not very opinionated on how you go about this.

Reply:
Ha no your way is much much better than my awful if statement.